src/blocks/recet.css

```diff
@@ -0,0 +1,223 @@
+html,

```

Comment:
you don't need all of this

Reply:
delete unused styles from `reset.css`. also fix typo in file name

Reply:
deleted unused styles

Reply:
delete unused styles from reset.css

Reply:
Why do you need recet.css and reset.css delete this file